src/bin/tikv-server.rs

```diff
@@ -71,8 +71,8 @@ fn get_string_value<F>(short: &str,
 fn initial_log(matches: &Matches, config: &toml::Value) {
     let level = get_string_value("L",
                                  "server.log-level",
-                                 &matches,
-                                 &config,
+                                 matches,
```

Comment:
okay...but I don't like PR with compile warning
this time it turn out to be a mistake with wrong clippy version